visitz/Visitz/Views/Todo/TodoMasterListViewModel.cs

```diff
@@ -1,23 +1,73 @@
+using System.Collections.ObjectModel;
+using System.ComponentModel;
 using CommunityToolkit.Mvvm.ComponentModel;
+using Realms;
+using Visitz.Storage;
 using Visitz.Views.BaseClasses;
-using VisitzModel.Interfaces;
-using VisitzModel.Models;
+using VisitzModel.Models.Drafts;
+using VisitzModel.Models.InPersonVisits;
 
 namespace Visitz.Views.Todo;
 
 public partial class TodoMasterListViewModel : VisitzViewModel
 {
     private bool _disposed;
 
+    [ObservableProperty]
+    public TodoItemUi selectedItem;
+    TodoItemUi upcomingVisitsItem;
+
+    [ObservableProperty]
+    ObservableCollection<object> todoMasterItems = [];
```

Comment:
```suggestion
    ObservableCollection<TodoItemUi> todoMasterItems = [];
```

See if this works, the bug that required object may have been fixed by MAUI.